index.html

```diff
@@ -3699,12 +3700,12 @@ <h2><span class="chunk">cICP</span> Coding-independent code points for video sig
             images</a>. If <code>Video Full Range Flag</code> value is <code>0</code>, then the image is a <a>narrow-range
             image</a>. Narrow range images are found in video workflows where the interpretation of sample values below reference
             black (0% signal level) or above nominal peak (100% signal level). For example, [[ITU-R-BT.709]] specifies that, for
-            10-bit coding, reference black (called black level) corresponds to code value 64 and nominal peak to code value 940.
-            In narrow range, momentary excursions defined as overshoots and undershoots exist below reference black and above nominal
-            peak in order to preserve processing artifacts caused by filtering/compression or by uncontrolled lighting without
-            clipping.  This can improve image quality during additional stages of processing and compression. The use of
-            undershoot/overshoot has also been used to preserve additional color volume (both light and color) as described in
-            [[EBU-R-103]]. [[SMPTE-RP-2077]] describes full range in more detail and includes the mapping
+            10-bit coding, reference black (called black level) corresponds to a 10-bit code value of 64 and nominal peak to a 
```

Comment:
There's no need to add "10-bit" here, as that's already established by the previous "for 10-bit coding"

Reply:
This looks fine.

Reply:
I think I've merged this